NitroxModel/Packets/GameModeChanged.cs

```diff
@@ -1,16 +1,32 @@
-﻿using System;
+using System;
 using NitroxModel.Server;
 
-namespace NitroxModel.Packets
+namespace NitroxModel.Packets;
+
+[Serializable]
+public class GameModeChanged : Packet
 {
-    [Serializable]
-    public class GameModeChanged : Packet
+    public ushort PlayerId { get; set; }
+    public bool AllPlayers { get; set; }
+    public NitroxGameMode GameMode { get; }
+
+    public GameModeChanged(ushort playerId, NitroxGameMode gameMode)
     {
-        public ServerGameMode GameMode { get; }
+        PlayerId = playerId;
+        GameMode = gameMode;
+    }
 
-        public GameModeChanged(ServerGameMode gameMode)
-        {
-            GameMode = gameMode;
-        }
+    public GameModeChanged(NitroxGameMode gameMode)
```

Comment:
Would change this to an static method so it's not mistaken for a "normal constructor"